include/crow/query_string.h

```diff
@@ -23,7 +23,7 @@ int qs_strncmp(const char* s, const char* qs, size_t n);
  *  Also decodes the value portion of the k/v pair *in-place*.  In a future
  *  enhancement it will also have a compile-time option of sorting qs_kv
  *  alphabetically by key.  */
-int qs_parse(char* qs, char* qs_kv[], int qs_kv_size, bool parse_url);
```

Comment:
I understand the reason behind this, but the maximum size here would be `256` so even an `unsigned char` would work here

Reply:
I think the signedness was the problem, not the size of the type. I used size_t because its nearly always a natural type of the processor (unsigned int or unsigned long int)